gltf-json/src/animation.rs

```diff
@@ -1,23 +1,25 @@
+use crate::validation::{Checked, Error, Validate};
+use crate::{accessor, extensions, scene, Extras, Index, Path, Root};
 use gltf_derive::Validate;
-use serde_derive::{Serialize, Deserialize};
 use serde::{de, ser};
+use serde_derive::{Deserialize, Serialize};
 use std::fmt;
-use crate::validation::{Checked, Error, Validate};
-use crate::{accessor, extensions, scene, Extras, Index, Path, Root};
 
 /// All valid animation interpolation algorithms.
+#[rustfmt::skip]
 pub const VALID_INTERPOLATIONS: &'static [&'static str] = &[
     "LINEAR",
     "STEP",
-    "CUBICSPLINE",
+    "CUBICSPLINE"
```

Comment:
Why are you removing the comma?

Reply:
Oops, rustfmt removed them, and I forgot to put them back. Anyways, I've remove the aestetics skips.